src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.figures;
+
+public class Square extends Rectangle {
```

Comment:
Interesting solutuion, but If you make inheritence like this Square will have 3 fields (side, firstSide and secondSide) if you want to keep this relation it will be better if you reverce it, so Rectangle will be extended from Square, in this case Rectange will inherit 1 field from Square and another field you will introduce yourself.